frontend/src/Keyword/Keyword.tsx

```diff
@@ -1,30 +1,31 @@
-import classes from './Keyword.module.css';
-import { PlusIcon, XMarkIcon } from '@heroicons/react/24/solid';
-import { KeywordOptions } from './KeywordTypes';
+import classes from "./Keyword.module.css";
+import { PlusIcon, XMarkIcon } from "@heroicons/react/24/solid";
+import { KeywordOptions } from "./KeywordTypes";
 
 type KeywordProps = {
   children: string;
-  type: KeywordOptions;
+  type?: KeywordOptions;
 };
 
-export function Keyword(props: KeywordProps) {
+export default function Keyword(props: KeywordProps) {
```

Comment:
so now you can:

`import Keyword from './Keyword'`

instead of

`import { Keyword } from './Keyword'`